### ISSUE TITLE
Free boxes in HUD 

### ISSUE THREAD

**Issue by PetrSvirak** [LindaLangerova/Furfur#22]
It is almost surely just me or you have plans with it, however, I don't understand why there is the Enter (with `carriage return` symbol) and 2 empty squares above the inventory in your HUD:
![image](https://user-images.githubusercontent.com/15132110/40262253-3d8a45d4-5b06-11e8-81ba-0f130d131523.png)

**Comment by LindaLangerova** [comment]
Yeah, I have plans with this :) However thanks, at least I will not forget about it.
